HHS/Head-Start-TTADP
# [TTAHUB-656] Goal Close/Suspend Reason Modal

## Description of change

If a user attempts to Close or Suspend a Goal we need them to provide a reason.

## How to test

Go to any recipient and view the 'Goals & Objectives' tab. When attempting to Close or Suspend a Goal you should be prompted to fill out a model with a reason and optional context.

**Note:** Before we push this PR the code to change the actual Goal status and reason will be _commented out_.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-656


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
